### ISSUE TITLE
Images not showing up in Aero's README

### ISSUE THREAD

**Issue by michaelrambeau** [michaelrambeau/bestofjs-webui#32]
If you open the project "Aero" http://bestof.js.org/projects/aero, images in the README.md section do not show up.

![image](https://cloud.githubusercontent.com/assets/5546996/20691869/55888d16-b617-11e6-9ec0-065d44500f27.png)

The cause seems to be the default Git branch that is called `nodejs`, there is no `master` branch.

URL that generates the README markup used by the application: https://get-github-readme-v1.now.sh/aerojs/aero

This URL returns the following path for the main screenshot:

https://github.com/aerojs/aero/raw/master/docs/images/aero-installation.png

It should be:

https://github.com/aerojs/aero/raw/nodejs/docs/images/aero-installation.png

(because image filepath in the repository is `/docs/images/aero-installation.png` and the branch name is `nodejs`)

I will fix the `get-github-readme` micro-service very soon to use the project default branch instead of `master`.

I also noticed that the table included in Aero README is not correctly styled.

![image](https://cloud.githubusercontent.com/assets/5546996/20691944/f81b5d7e-b617-11e6-8700-9aa32ee611cc.png)

Thank you @blitzprog for having introduced Aero project to me!

**Comment by blitzprog** [comment]
Sorry, I'm really busy because of my [next project](https://github.com/aerogo) so I haven't found the time to properly respond.
I just want to say thank you for including aero for node.js on the site and keep up the good work!
If I ever dive into JS again for a future project I'll look at your site :)
Looking forward to see you again at events in the Kansai area.

**Comment by michaelrambeau** [comment]
Hello @blitzprog 
Just to confirm, I have fixed the problem in the README.md and improved the styling.
You can check here => http://bestof.js.org/projects/aero

Thank you again, and if you are in Kyoto Kawaramachi area, I'd be more than happy to discuss with you during lunch time, we had such a little time last Saturday.
I cannot believe that I sat down near the creator of the fastest node.js framework in the world :)
I may contribute to your project very soon (to add Loopback to the benchmark part for example).

And by the way, let me know if you feel there is room to create something like **bestofgolang.com**!

**Comment by blitzprog** [comment]
Nice, the table styling look good :)

Do you know about the "awesome" lists?

https://github.com/sindresorhus/awesome
https://github.com/avelino/awesome-go

I use the Go list every now and then to check on new libraries.
I don't think someone made a website for exploring those (yet).

**Comment by michaelrambeau** [comment]
Hello Eduard, thank you for having checked.
Yes, I know the `awesome` project... it's the 5th most popular project on bestof.js.org!

Over the last months, I noticed that people use more and more simple markdown files stored on Github to share content, whether it's documentation or lists of interesting things.
Even if it's community driven, it's hard to know whether the links added one day are still relevant 6 months later, the landscape changes so fast.
This is one of the reasons I created bestof.js.org, I wanted to see not only the "popular" projects but also the ones that have traction (the ones that get new stars everyday on Github).
